.github/ISSUE_TEMPLATE/85_bug-report.md

```diff
@@ -7,6 +7,8 @@ assignees: ''
 
 ---
 
+> Please make sure that the version you're using is still supported. The rules described [here](https://clickhouse.com/docs/en/faq/operations/production#how-to-choose-between-clickhouse-releases).
```

Comment:
We also have a list of supported versions in [SECURITY.md](https://github.com/ClickHouse/ClickHouse/blob/master/SECURITY.md#scope-and-supported-versions), which is more straightforward to check